sdk/react/index.android.js

```diff
@@ -57,6 +57,8 @@ var OoyalaSkin = React.createClass({
       // things which default to null and thus don't have to be stated:
       // selectedLanguage: null,
       // availableClosedCaptionsLanguages: null,
+      // selectedAudioTrack,
```

Comment:
can you add `multiAudioEnabled: false` to the `getInitialState` function? So that we can see that at the start it is clear that multiAudio is not enabled.